src/apis/getChannels.ts

```diff
@@ -1,4 +1,9 @@
 // 채널 목록 조회
+import { api } from '@/apis/core';
 import { Channel } from './responseModel';
 
 export type Channels = Channel[];
+
+export const getChannels = api.get<Channels>({
```

Comment:
```suggestion
export const getChannels = () =>  api.get<Channels>({
```

로 해야하지 않을까요 ?? 

그리고 getChannels가 사용되고있지 않은데 그부분도 한번 체크해보시면 좋을 것 같아요

Reply:
지금 채널목록은 새로 텐스택에서 꺼내온 api가 아니라 RTK 에서 데이터가 얻어와지고 있네요 :) 체크 해보시면 좋을 것 같습니다

Reply:
아하 제가 pr 이 너무 커지는것 같아서, 일단 post list 부분만 tanstack query 로 바꾸는 작업을 했는데,
이건 하다가 애매하게 놔둬버렸네요..

일단, 해주신대로 나중에 쓰일수있게 api를 수정 해놓겠습니다.